cmd/solution/upload.go

```diff
@@ -211,6 +210,9 @@ func uploadSolution(cmd *cobra.Command, push bool) {
 	}
 
 	if subscribe, _ := cmd.Flags().GetBool("subscribe"); subscribe {
+		if solutionTagFlag != "stable" {
+			solutionName += ".dev"
```

Comment:
1. I would propose using a separate variable for the subscribe name, so that we don't clobber the name for anyone who wants to use it after the subscribe block. Note that the "wait" block below also needs to use this name, so I would propose something like `solutionObjName` (and update the wait to use it as well, calculate it always, not only when --subscribe). [The reason I propose `solutionObjName` is that this is the object name under which the solution object will be found; it is used for all operations that read the solution object (at tenant layer) as opposed to doing a specific REST call (like upload).
2. There is one more step needed here: if solution isolation is enabled (which we need to mark/remember if line 116 `if` statement is true), then we need to add the requestedTag as well: 
    `solutionObjName += requestedTag + ".dev"`

Reply:
for number 2, Im pretty sure that solution name already contains the relevant isolation information